data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+				has "Remnant Tech Retrieval: done"
+			`	After asking a few people, you finally get directed to a bay in the shipyard where you find Taely closing up the panel on a thasher turret. "Greetings, <first>. You have good timing: I was just finishing." She reseals the hatch, and drops to the ground on a tether. "So, did you recover something?"`
+				goto the_rest
+			label stuff
+			`	Remembering that she spends most of her time in the shipyard, you head straight there. After a frustrating half hour spent wandering around the spaceport tryping to find her, you finally encounter a helpful weapons tech who says they can check the weapon on Taely's behalf.`
+			label the_rest
+			`	You accompany her back to your ship, where she examines the electron beam. "This could be useful. Focusing energy into continuous beams has been a popular idea for centuries. It is very useful to know that they have finally managed to make it into an effective weapon.`
+				accept
+	on complete
+		outfit "Electron Beam" -2
+		payment 1090000
+		"remnant met taely" ++
+		conversation
+			`On <planet> you follow the landing instructions to arrive at a pad set next to a small complex of testing ranges and workshops. A group of technicians is just finishing the disassembly of a weapon from the test stand when you settle onto the stand, and a few minutes later they arrive to unload the electron beams from your ship. They move eagerly to secure the new weapons to their flatbed and quickly hand you your payment of <payment> before giving vanishing into the lab.`
 
-			`	"Don't worry," he sings with a tinge of reproach. "I am a Remnant, not a Korath."`
-			`	He proceeds to methodically examine your ship from every angle on the ground, eventually focusing on your weapon ports and hardpoints. After examining the <ship> for quite some time, he walks over to you.`
-				goto more
 
-			label deep
-			`	He clips his harness to one of the overhead lifts, presses a button on the back of his gauntlet, and is promptly whisked up into the air. The lift appears designed to give engineers and technicians almost as much freedom as a zero-g environment by suspending them with just enough force to counteract gravity, so they can move around a ship with ease. Having completed a quick once-over he quickly gravitates to the weapon hardpoints. After examining the <ship> for quite some time, he calls down to you.`
 
-			label more
-			`	"I have never seen weapons designed to handle such a high heat output! Is this some kind of plasma based weapon?" he asks, gesturing at a plasma cannon. You nod, and he continues, "Intriguing. We have noticed that the Korath ships seem to run particularly hot, and have speculated that overheating them might be an effective means of disabling them. I would enjoy researching these a bit more. If you could acquire some for us you would be well compensated."`
+mission "Remnant: D94-YV Shield Generator"
+	name "Retrieve D94-YV Shield Generators for the Remnant"
```

Comment:
Name looks dangerously long.

Reply:
Removed the `for the Remnant` part to shorten it up.